### PR TITLE
DT_THERM=6*DT and DIABATIC_FIRST = False

### PR DESCRIPTION
**Pull request by aekiss**
use DT_THERM=6*DT and DIABATIC_FIRST = False - see https://github.com/COSIMA/ACCESS-OM3/issues/138

### CODE REVIEW

File: nuopc.runconfig [hunk 1]
```diff
@@ -286,15 +286,15 @@ CLOCK_attributes::
...
+     ocn_cpl_dt =  1800
      restart_n = 1
      restart_option = nmonths
      restart_ymd = -999
      rof_cpl_dt = 99999 #not used
      start_tod = 0
      start_ymd = 19000101
      stop_n = 1
-     stop_option = nmonths
+     stop_option = nyears
```
Comment (aekiss) [inline nuopc.runconfig:297]:
```suggestion
     stop_option = nmonths
```

File: config.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@ ncpus: 48
 jobfs: 10GB
 mem: 192GB
 
-walltime: 01:00:00
+walltime: 04:00:00
```
Comment (aekiss) [inline config.yaml:16]:
```suggestion
walltime: 01:00:00
```

File: nuopc.runconfig [hunk 3]
```diff
@@ -286,7 +286,7 @@ CLOCK_attributes::
      history_ymd = -999
      ice_cpl_dt = 99999 #not used
      lnd_cpl_dt = 99999 #not used
-     ocn_cpl_dt =  3600
+     ocn_cpl_dt =  1800
```
Comment (minghangli-uni) [inline nuopc.runconfig:289]: It is not necessarily to change the value of `ocn_cpl_dt` unless `stop_option` is `nsteps`. I can do a cherry pick to add comments right after this parameter for all configurations. https://github.com/COSIMA/access-om3/issues/157#issuecomment-2249623984

**Review by minghangli-uni — APPROVED** [review]
Thanks @aekiss LGTM!

**Comment by aekiss** [comment]
!cherry-pick ac53042 c18a988 into 1deg_jra55do_iaf

**Comment by aekiss** [comment]
!cherry-pick 58e3b755e8e09664f4c8eedb40f428ce215a4bfb 2d639fe009f92860169677b18990864317f0a9ca into 1deg_jra55do_iaf

**Comment by minghangli-uni** [comment]
Hi @aekiss It seems that your cherry-picks failed because of the conflicts. You might need to follow the steps from https://github.com/COSIMA/MOM6-CICE6/pull/94#issuecomment-2272702554

**Comment by aekiss** [comment]
OK I'm working on it. The biggest conflict is due to differing `MOM_input` formats - see https://github.com/COSIMA/access-om3/issues/200

**Comment by dougiesquire** [comment]
I was under the impression we weren't going to change these params in our official config branches until we'd done the testing to determine what `DT_THERM` is optimal. Are we confident this value gives okay results?

**Comment by aekiss** [comment]
Ah yes, good point. Sorry if I jumped the gun there. I'll wait until we've tested before merging this into `1deg_jra55do_iaf`.

**Pull request merged**